roles/infrastructure/tasks/setup_aws_network.yml

```diff
@@ -40,29 +40,27 @@
   ansible.builtin.set_fact:
     infra__aws_igw_id: "{{ __aws_igw.gateway_id }}"
 
-- name: Create AWS VPC Subnets
+- name: Create AWS VPC Public Subnets
   amazon.aws.ec2_vpc_subnet:
     region: "{{ infra__region }}"
     vpc_id: "{{ infra__aws_vpc_id }}"
-    cidr: "{{ __aws_subnet_create_item.cidr }}"
+    cidr: "{{ __aws_subnet_cidr_item }}"
     state: present
-    tags: "{{ infra__tags | combine(__aws_subnet_create_item.tags, recursive=True) }}"
+    tags: "{{ infra__tags | combine({ 'Name': '-'.join([infra__namespace, 'pub', __aws_subnet_index | string]) }, recursive=True) }}"
```

Comment:
I would prefer if we didn't use a static 'pub' here and used a label variable like the others

Reply:
let me fix this